src/chat/chatter.py

```diff
@@ -295,11 +296,13 @@ def pressed(self, item):
                 action_inspect_in_mordor = QtGui.QAction("Send the Orcs", menu)
                 menu.addAction(action_inspect_in_mordor)
 
+                ROOT = Settings.get('mordor/host')
+
                 def send_the_orcs():
                     if self.id != -1:
-                        QtGui.QDesktopServices.openUrl(QUrl("http://mordor.faforever.com/users/{}".format(self.id)))
+                        QtGui.QDesktopServices.openUrl(QUrl("http://" + ROOT + "/users/{}".format(self.id)))
```

Comment:
I guess this means you don't know how the format function works. It basically does substitutions: each argument to the format function replaces the corresponding "{}" in the string.

You can do an even neater thing in python:

`"Here are some numbers: %s, %s, %s" % (1, 2, 3)`

Is equivalent to `"Here are some numbers: 1, 2, 3"`.

You might want to do it that way here.


That said, an even better way is to hide this `ROOT` rubbish. Why does every single usage of any of the FAF web routes have to do this string building itself? That's annoying. Wouldn't it be nice to be able to do something like:

`QtGui.QDesktopServices.openUrl(getRoute("/users/%s", self.id))`

That should be rather obviously doable, I hope?